lib/rialto/etl/cli/grants.rb

```diff
@@ -97,7 +97,8 @@ def extract(row, profile_id, force)
           end
           # Don't write file if an exception occurs
           begin
-            results = perform_extract(row)
+            return extract_file if row[:sunetid].empty? # return from extract without calling SERA API
```

Comment:
@aaron-collier Cool! Can you confirm that the weirdo `""` values in the CSV are parsed into Ruby as things for which `#empty?` is `true`?

Reply:
Yes, confirmed.